guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc

```diff
@@ -24,7 +23,8 @@ endif::[]
 . Select a {customproduct} by name.
 . Select a file type repository by name.
 . Ensure to select the *Unprotected* checkbox to access the repository published through HTTP.
-. Copy the URL where the repository is published.
+. Copy the *Published At* URL.
+include::snip_downloading-file-from-server.adoc[]
```

Comment:
```suggestion
include::snip_step-downloading-file-from-server.adoc[]
```
I'd add the "snippet type" to the file name.

Reply:
Renamed.